_targets.R

```diff
@@ -24,19 +26,29 @@ major_ion_names = c("Chloride","Sodium")
 # Define hydrologic event types in harmonized WQP data to exclude
 omit_wqp_events <- c("Spill","Volcanic action")
 
-# Define USGS specific conductance parameter codes
-SpC_pcodes <- c("00095","90095","00094","90096") 
+# Define all USGS specific conductance parameter codes
+pcodes_select <- c("00095","90095","00094","90096") 
```

Comment:
Suggestion: Since most of these variables are specific to either `1_fetch.R`, `2_process.R`, it would be easier to have them defined within the phase target scripts rather than in the overarching `_targets.R` if that is possible.

Reply:
I can definitely see how keeping the variables within their respective phase files would be easier to read. I think we initially even started that way but at one point we decided to keep all of the defined variables within the `_targets.R` file so that a user would see them all in one place, as opposed to having to check the individual phase files. I'll suggest we leave as is for now, but we can revisit this sort of formatting if our `_targets.R` file starts to get too unwieldy!